pkg/cmd/roachtest/restore.go

```diff
@@ -306,17 +306,36 @@ func registerRestoreNodeShutdown(r *testRegistry) {
 }
 
 func registerRestore(r *testRegistry) {
+	largeVolumeSize := 2500 // the size in GB of disks in large volume configs
+
 	for _, item := range []struct {
-		nodes   int
+		nodes        int
+		cpus         int
+		largeVolumes bool
+
 		timeout time.Duration
+		tags    []string
 	}{
-		{10, 6 * time.Hour},
-		{32, 3 * time.Hour},
+		{nodes: 10, timeout: 6 * time.Hour},
+		{nodes: 32, timeout: 3 * time.Hour},
+		// Run a cluster with larger nodes weekly.
+		{nodes: 12, timeout: 3 * time.Hour, cpus: 16, largeVolumes: true, tags: []string{"weekly"}},
```

Comment:
I'd do at least nodes=6 nightly, unless it costs _way_ more for some reason?

Reply:
Defaulted to weekly just in case, since I figured these will mostly be run by one-off benchmarking for now. But looked at https://cloud.google.com/compute/all-pricing#persistentdisk and it doesn't look like it'd be too much more. Will update to 6 nodes nightly.

Updated.